src/Microsoft.OData.Core/Parameterized.Microsoft.OData.Core.cs

```diff
@@ -2147,6 +2147,13 @@ internal static string ReaderValidationUtils_NullValueForNonNullableType(object
             return Microsoft.OData.TextRes.GetString(Microsoft.OData.TextRes.ReaderValidationUtils_NullValueForNonNullableType, p0);
         }
 
+        /// <summary>
+        /// A string like "A null value was found with the expected type '{0}[Nullable=True]'. The expected type '{0}[Nullable=True]' does not allow null values."

```

Comment:
I was trying to understand the difference between this and ReaderValidationUtils_NullValueForNonNullableType, but the only difference is whether or not the type is nullable (and it wasn't clear from this message why a nullable type wouldn't allow null values). please update to match the string in Microsoft.OData.Core.txt.

Reply:
resolved